ipa/scripts/cli.py

```diff
@@ -147,6 +147,11 @@ def main():
     help='Remove ANSI color and styling from output.'
 )
 @click.option(
+    '--no-default-test-dirs',
+    is_flag=True,
+    help='Do not include default test directories in test run.'
```

Comment:
Is there an option in Click to set a default value? If yes, it should be set to "False" so the reader immediately knows the intend

Reply:
Default for flags is False but updated for readability.